plugins/org.yakindu.sct.model.sexec/src/org/yakindu/sct/model/sexec/transformation/ng/ModelSequencer.xtend

```diff
@@ -119,10 +119,10 @@ class ModelSequencer implements IModelSequencer {
 		
 		eventProcessing.defineFeatures(ef)
 		executionGuard.defineFeatures(ef)
-		enterMethod.defineFeatures(ef)
-		exitMethod.defineFeatures(ef)
 		superStep.defineFeatures(ef)
 		runCycleMethod.defineFeatures(ef)
+		enterMethod.defineFeatures(ef)
+		exitMethod.defineFeatures(ef)
```

Comment:
With this change, the order of methods also changes in the generated code, i.e. enter/exit methods will be defined after runCycle method.

I had to do it this way because RunCycleMethod does not provide a create method.

Reply:
I added a create method now for the microStep